project_budget_management/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Project Budget Management",
+    "version": "15.0.1.0.0",
+    "summary": "A Module adds features to the Project Budget Management.",
+    "author": """Serpent Consulting Services Pvt. Ltd.""",
+    "website": "http://www.serpentcs.com",
+    "category": "Project Budget Management",
+    "license": "LGPL-3",
+    "depends": ["project_timesheet_management"],
```

Comment:
Where is this module?

Reply:
Will add pr of timesheet module today, but kindly note that it still require code level improvements as its WIP.

Reply:
ok make sure all the base modules are added before add other modules